eiffel-vocabulary/EiffelActivityQueuedEvent.md

```diff
@@ -22,8 +22,13 @@ __Description:__ The circumstances triggering the activity.
 #### data.trigger.type
 __Type:__ String  
 __Required:__ Yes  
-__Legal values:__ MANUAL, EIFFEL_EVENT, REBUILD, SOURCE_CHANGE, TIMER, OTHER  
-__Description:__ The type of trigger.
+__Legal values:__ MANUAL, EIFFEL_EVENT, SOURCE_CHANGE, TIMER, OTHER  
```

Comment:
A question here; what about the activities that are queued due to multiple different types of triggers? (This could happen if a job is not allowed to run concurrently and put into queue and mergeable queue is used - sorry for thinking in Jenkins terms.)